CHANGELOG_UNRELEASED.md

```diff
@@ -12,12 +12,71 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Changed
 
+- Notations declared in the `fun_scope` are now declared in the
+  `function_scope`.
+
+- in `finset.v`
```

Comment:
CHANGELOG contains some entries from other PRs.

Reply:
Ooops, missed that, fixed in https://github.com/math-comp/math-comp/pull/1135